presto-main/src/test/java/com/facebook/presto/block/AbstractTestBlock.java

```diff
@@ -88,6 +88,10 @@ private static long getSliceArrayRetainedSizeInBytes(Slice[] values)
             if (value.getBase() != null && uniqueRetained.put(value.getBase(), true) == null) {
                 sizeInBytes += value.getRetainedSize();
             }
+            else if (value != EMPTY_SLICE) {
+                // EMPTY_SLICE is a singleton in the system
```

Comment:
You can expand on this comment for the future reader of this code, something along the lines of:
`EMPTY_SLICE is a singleton, so we don't account for the memory held onto by that instance. Otherwise, we will be counting it multiple times.`